src/config.js

```diff
@@ -255,7 +255,7 @@ export const config = configOrder({
 		properties: {
 			webgl: {
 				title: 'WebGL Renderer',
-				description: 'Enable the WebGL-based renderer using the xterm.js [WebGL addon](https://github.com/xtermjs/xterm.js/tree/master/addons/xterm-addon-webgl)',
+				description: 'Enable the WebGL-based renderer using the xterm.js [WebGL addon](https://github.com/xtermjs/xterm.js/tree/master/addons/xterm-addon-webgl) ⚠️[Experimental](https://github.com/xtermjs/xterm.js/tree/master/addons/xterm-addon-webgl#xterm-addon-webgl)⚠️',
```

Comment:
```suggestion
				description: '⚠️ Enable the Experimental WebGL-based renderer using the xterm.js [WebGL addon](https://github.com/xtermjs/xterm.js/tree/master/addons/xterm-addon-webgl) ',
```

No need for two links pointing to same addon age, and definitely not as noticeable at the end.

Reply:
I feel like having the icon in the front or around the whole string doesn't let the reader know which part they should be concerned about. Should they be concerned that they are enabling something or that it is WebGL-based?

how about:
![image](https://user-images.githubusercontent.com/97994/91193483-c3b6d200-e6bc-11ea-947e-e6cd84e2f22b.png)

or

![image](https://user-images.githubusercontent.com/97994/91193677-fcef4200-e6bc-11ea-82e6-949d91b53874.png)

Reply:
I updated it to put it in the title

![image](https://user-images.githubusercontent.com/97994/91193677-fcef4200-e6bc-11ea-82e6-949d91b53874.png)